lsp-ui-sideline.el

```diff
@@ -296,11 +303,10 @@ CURRENT is non-nil when the point is on the symbol."
          (str (if lsp-ui-sideline-show-symbol
                   (concat info " " (propertize (concat " " symbol " ") 'face face))
                 info))
-         (len (length str))
-         (margin (lsp-ui-sideline--margin-width)))
+         (len (length str)))
     (add-face-text-property 0 len 'lsp-ui-sideline-global nil str)
     (concat
-     (propertize " " 'display `(space :align-to (- right-fringe ,(lsp-ui-sideline--align len margin))))
+     (propertize " " 'display `(space :align-to (- right-fringe ,(lsp-ui-sideline--align len))))
```

Comment:
Now we have the calculation in function `lsp-ui-sideline--margin-width`. Remove the extra margin.